packages/manager/apps/pci-private-registry/src/components/Plan.component.spec.tsx

```diff
@@ -6,21 +6,48 @@ import { TCatalog } from '@ovh-ux/manager-pci-common';
 import PlanComponent from './Plan.component';
 import { wrapper } from '@/wrapperRenders';
 import { TRegistryPlan } from '@/api/data/registry';
+import { useStore } from '@/pages/create/store';
+import { DeploymentMode, PlanName } from '@/types';
 
-const defaultPlan: TRegistryPlan = {
-  id: 'planId',
-  code: 'planCode.hour.consumption',
-  features: {
-    vulnerability: false,
-  },
-  name: 'SMALL',
-  registryLimits: {
-    imageStorage: 10,
-    parallelRequest: 50,
-  },
-  createdAt: '',
-  updatedAt: '',
-};
+vi.mock('@/pages/create/store', () => ({
+  ...vi.importActual('@/pages/create/store'),
+  useStore: vi.fn(),
+}));
+
+vi.mock('react-i18next', () => ({
+  useTranslation: vi.fn().mockImplementation(() => ({
+    t(key: string, data: Record<string, string>) {
+      const templates: Record<
+        string,
+        (data: Record<string, string>) => string
+      > = {
+        private_registry_upgrade_plan_connections: (d) =>
+          `private_registry_upgrade_plan_connections:${d.total}`,
+        private_registry_upgrade_core_registry: (d) =>
+          `private_registry_upgrade_core_registry_${d.percent}`,
+        private_registry_upgrade_other_components: (d) =>
+          `private_registry_upgrade_other_components_${d.percent}`,
+      };
+
+      return templates[key]?.(data) ?? key;
+    },
+  })),
+}));
+
+vi.mock('@ovh-ux/manager-react-components', async () => ({
+  useMe: () => ({
+    me: {
+      ovhSubsidiary: 'ovhSubsidiary',
+      currency: {
+        code: 'USD',
+      },
+    },
+  }),
+  useCatalogPrice: () => ({
+    getFormattedCatalogPrice: vi.fn(),
+  }),
+  convertHourlyPriceToMonthly: vi.fn(),
+}));
```

Comment:
I think this could be in global setupTests

Also the async doesn't do anything here